docs/INDEX.md

```diff
@@ -1,94 +1,84 @@
 ---
 home: true
+heroImage: /logo.svg
 actionText: Get Started  →
 actionLink: /docs/latest
-
-footer: Copyright © 2019 Christopher Moore
+features:
+- title: 📜 SDL First
```

Comment:
I don't think we should use emojis directly here as they render differently on different operating systems.
Here is an example of mine, which is different from what you sent.
![image](https://user-images.githubusercontent.com/5870441/66374425-42bef100-e9ab-11e9-9a7c-6ac41f6a66b7.png)

Reply:
That might be seen as a strength too. Everybody gets the emoji's that they are used to from their system.

Reply:
Yeah, but if you look at the image you sent of this, the first emoji did not render on your machine

Reply:
Emojis are universally used, and i don't think it is a big deal if they are not displayed in some rare cases.

You used one in this very comment chain 😉 I think we are good here and really don't feel like including custom icons for every place where we use emojis.

Reply:
Sure thing 👍 Just wanted to make you aware of it.